docs/pages/build/internal-distribution.md

```diff
@@ -44,7 +48,13 @@ Open up `eas.json` and add a new build profile for iOS and/or Android.
 }
 ```
 
-> If you override the `gradleCommand` on Android, you should ensure that it produces an `apk` rather than an `aab`, so it is directly installable to an Android device.
+### Android
+
+Please note that if you override the `gradleCommand` on Android, you should ensure that it produces an `apk` rather than an `aab`, so it is directly installable to an Android device.
+
+### iOS
+
+Add the `preview-enterprise` profile only if you have an Apple account with Apple Developer Enterprise Program membership. You'll learn more on this later. While using the enterprise provisioning, you can sign your app using a `universal` or `adhoc` provisioning profile. The former (`universal`) is recommended as it does not require you to register your devices with Apple.
```

Comment:
`You'll learn more on this later` -> `You can learn more about this [here](some url)
or just remove that sentence